xbmc/video/VideoDatabase.cpp

```diff
@@ -1025,6 +1025,34 @@ bool CVideoDatabase::GetLinksToTvShow(int idMovie, std::vector<int>& ids)
   return false;
 }
 
+bool CVideoDatabase::GetLinksToTvShow(std::map<int, CVideoInfoTag>& details)
```

Comment:
It was like this then I changed to one parameter. It makes sense.. I'll change it back